easybuild/easyblocks/a/anaconda.py

```diff
@@ -42,33 +42,11 @@
 
 def set_conda_env(installdir):
     """ Set the correct environmental variables for conda """
-    myEnv = os.environ.copy()
-    env.setvar('PATH', "{}/bin".format(installdir) + ":" + myEnv["PATH"])
+    env.setvar('PATH', os.path.join(installdir, 'bin') + ':' + os.environ.get('PATH', ''))
```

Comment:
OK, it seems it depends on the Python version you're using; I was testing with Python 2.6.x, where `{}` is not supported yet, see http://stackoverflow.com/a/8498327/1467322